.github/workflows/ci.yml

```diff
@@ -76,10 +76,10 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Make .sh executable
-        run: chmod +x bash_scripts/prevent_absolute_links_to_docs.sh
+        run: chmod +x scripts/prevent_absolute_links_to_docs.sh
```

Comment:
don't know why I didn't see this before, but if you make the script executable locally and then commit it, you don't have to do this.